### ISSUE TITLE
Magento 2.3.3: symfony/service-contracts dependency is defined twice in the composer.lock

### ISSUE THREAD

**Issue by jorgb90** [magento/magento2#24937]
### Preconditions (*)
1. Magento 2.3.3

### Additional details
1. detailed explanation https://github.com/magento/magento2/issues/24937#issuecomment-540164906
2. https://github.com/magento/magento2/blob/2.3.3/composer.lock#L9196
https://github.com/magento/magento2/blob/2.3.3/composer.lock#L9254

### Steps to reproduce (*)
1. Download Magento 2.3.3
-- as arch from https://github.com/magento/magento2/releases/tag/2.3.3
-- or
 ```
git clone https://github.com/magento/magento2.git
git checkout tags/2.3.3
```
2. Run composer install

### Expected result (*)
1. Installing packages

### Actual result (*)
1. /public$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package symfony/service-contracts v1.1.6 is satisfiable by symfony/service-contracts[v1.1.6] but these conflict with your requirements or minimum-stability.

**Comment by milindsingh** [comment]
I am closing this issue as I am not able to reproduce it on vanilla 2.3.3 on Php 7.2.
Please, can you check the 3rd party module dependencies for any conflicts.
Thanks

**Comment by jorgb90** [comment]
@milindsingh There are no 3rd party modules.. I just download the 2.3.3 zip release from GitHub, unzip it and run composer install. In a clean directory of course.

**Comment by hostep** [comment]
@jorgb90: I can confirm this is indeed an issue when downloading this from the GitHub releases.
It is however not a problem when setting up Magento using composer or when downloading the archive from https://magento.com/tech-resources/download#download2321, which is a more recommended way of installing Magento.

The problem with the GitHub release is that for some bizarro reason, the `symfony/service-contracts` dependency is defined twice in the `composer.lock` file each time with a different version, which is really really strange.

<img width="615" alt="Screenshot 2019-10-09 at 21 55 03" src="https://user-images.githubusercontent.com/85479/66515611-b3672a00-eadf-11e9-9f9a-c839768141d7.png">

However, to fix your problem, you can simply run `composer update` instead of `composer install` which will remove the duplicated `symfony/service-contracts` dependency from the `composer.lock` file and also update some other dependencies.

*Update*: you can also just run `composer update symfony/service-contracts` which also seems to solve the problem, but then without updating the other dependencies.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @sdzhepa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-21670` were created

**Issue Available**: @sdzhepa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by jorgb90** [comment]
@hostep Thanks for looking into it!

**Comment by vivekkumar945744** [comment]
@magento I am working on it.

**Comment by krissyhiserote** [comment]
This should already be fixed and delivered into 2.3-develop.
